src/less/techne-patterns/_locals-editor.less

```diff
@@ -190,28 +140,26 @@
 //                <li> <a href="">DA</a></li>
 //                <li> <a href="">NL</a></li>
 //                <li class="dropdown locals-tabs__more-menu active">
-//                    <a class="dropdown-toggle is-required locals-tabs__more-btn" data-toggle="dropdown" aria-expanded="false">IT
+//                    <a class="dropdown-toggle has-error locals-tabs__more-btn" data-toggle="dropdown" aria-expanded="false">IT
```

Comment:
I just reviewed it with dieter. he said to change this to "More" in this case since we do not show an example with error on tab within the more menu.